client/classic/upe/deferred-intent.js

```diff
@@ -41,6 +41,22 @@ jQuery( function ( $ ) {
 	} );
 
 	$( 'form.checkout' ).on( generateCheckoutEventNames(), function () {
+		const $form = $( 'form.checkout' );
+		// Lose focus for all fields to trigger validation
+		$form
+			.find( '.input-text, select, input:checkbox' )
+			.trigger( 'validate' )
+			.trigger( 'blur' );
```

Comment:
In shortcode checkout, the validation check is done after the `/checkout` request is sent. `createPaymentMethod` is called before the `/checkout` request.
So we have to explicitly trigger the validation check here as we want to prevent `createPaymentMethod` call based on the validation result.